build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
 		url 'http://jastadd.org/mvn/'
 	}
 	dependencies {
-		classpath group: 'org.jastadd', name: 'jastaddgradle', version: '1.9.4'
+		classpath group: 'org.jastadd', name: 'jastaddgradle', version: '1.9.6'
```

Comment:
Version 1.9.9 is the latest version, and the preferred version because it uses only Maven Central for all dependencies (instead of relying on jastadd.org).